ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -208,6 +208,7 @@ objects:
             elif [[ -z "${API_UP}" ]]; then
               echo "API at $(oc whoami --show-server) has responded"
               API_UP=1
+              break
```

Comment:
This break is required otherwise while loop never stopped till the timeout occur.

Reply:
:+1:

Reply:
Do we still need `cluster_version_check()` at all now that the installer blocks until the CVO reports success?

Reply:
@wking done, updated.